packages/config/config/devices/0x010f/fgd212_3.5.json

```
@@ -167,315 +173,383 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "No calibration or manual settings",
+					"label": "No Calibration or Manual Settings",
 					"value": 0
 				},
 				{
-					"label": "Dimmer operates on auto-calibration settings",
+					"label": "Dimmer Operates on Auto-calibration",
 					"value": 1
 				}
 			]
 		},
 		"15": {
-			"label": "Burnt out bulb detection",
-			"description": "Percentage value of power variation",
+			"label": "Burnt Out Bulb Detection Power Variation (LOAD ERROR)",
+			"description": "Percentage value of standard power consumption as measured during calibration",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 99,
-			"defaultValue": 30
+			"defaultValue": 30,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		"16": {
-			"label": "Time delay of a burnt out bulb",
-			"description": "Time of delay for power variation detection",
+			"label": "Burnt Out Bulb Detection Time Delay (LOAD ERROR)",
 			"valueSize": 2,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 5
+			"defaultValue": 5,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		"19": {
-			"label": "Forced switch on brightness level",
-			"description": "Dimmer 2 will always set this brightness level.",
+			"label": "Forced Switch On Brightness Level",
+			"description": "Percentage value.",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 99,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disable, Previous Value is Used",
+					"value": 0
+				}
+			]
 		},
 		"20": {
-			"label": "Inputs Button/Switch configuration",
-			"description": "Binary inputs type configuration",
+			"label": "Input Button/Switch Configuration",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 2,
 			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Mono-stable input (button)",
+					"label": "Mono-stable Input (Button)",
 					"value": 0
 				},
 				{
-					"label": "Bi-stable input (switch)",
+					"label": "Bi-stable Input (Switch)",
 					"value": 1
 				},
 				{
-					"label": "Roller blind switch (UP / DOWN)",
+					"label": "Roller Blind Switch (UP / DOWN)",
 					"value": 2
 				}
 			]
 		},
 		"21": {
-			"label": "Value sent to associated devices on single click",
+			"label": "Value Sent to Associated Devices on Single Click",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 1,
 			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Set associated devices to last state",
+					"label": "Set Associated Devices to Last State",
 					"value": 0
 				},
 				{
-					"label": "Dimmer 2 state synchronizes brightness level",
+					"label": "This Dimmers Current State Synchronizes Brightness Level",
```

The previous label confused me, what about this?
```suggestion
					"label": "This dimmer's current brightness",
```